### PR TITLE
add documentation project

### PR DESCRIPTION
**Pull request by nslater**

### CODE REVIEW

File: docs/getting-started.txt [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+Prerequisites
+=============
+
+.. CAUTION::
+
+    For now, this plugin depends on a `fork`_ of the official `Npgsql
+    project`_ to work. We are, however, working to get the necessary changes
+    merged in upstream.
+
+For convenience, we provide a forked `Npgsql installer`_ that you can download
```
Comment (mikethebeer) [inline docs/getting-started.txt]: Mention that the MSI Installer (=GAC installer) is **NOT** the recommended way. It is only necessary for 3rd party apps like Microsoft PowerBI. In some cases you'll want to install Npgsql into your Global Assembly Cache (GAC). This is usually the case when you're using a generic database program that can work with any ADO.NET provider but doesn't come with Npgsql or reference it directly. For these cases, you can download the Npgsql Windows installer from `cnd.crate.io`

Comment (mikethebeer) [reply docs/getting-started.txt]: Check http://www.npgsql.org/doc/index.html#gac-installation

**Comment by nslater** [comment]
@mikethebeer I have addressed your comment. please re-review

File: docs/getting-started.txt [hunk 2]
```diff
@@ -45,6 +48,7 @@ Once the plugin is installed, you probably want to :ref:`connect to CrateDB
 .. _dotnet CLI: https://docs.microsoft.com/en-us/nuget/quickstart/install-and-use-a-package-using-the-dotnet-cli
 .. _fork: https://github.com/crate/npgsql
 .. _introduction to Nuget: https://docs.microsoft.com/en-us/nuget/what-is-nuget
-.. _Npgsql installer: https://cdn.crate.io/downloads/releases/npgsql/
+.. _MSI Installer: https://cdn.crate.io/downloads/releases/npgsql/
```
Comment (mikethebeer) [inline docs/getting-started.txt]: Since `MSI` (Microsoft Installer) means already "Installer" I would stay with "Npgsql Installer" 😄

**Pull request merged**